src/gennylib/src/PoolManager.cpp

```diff
@@ -74,24 +74,34 @@ mongocxx::pool::entry genny::v1::PoolManager::client(const std::string& name,
 
     Pools& pools = lap.second;
 
+    auto& clientNode = workloadCtx["Clients"][name];
+    bool shouldPrewarm = !_dryRun & !clientNode["NoPreWarm"].maybe<bool>().value_or(false);
```

Comment:
nit: `&` ==> `&&` ? Although `&` gives correct result, we are doing an "logical and" and hence I'd prefer the logical operator `&&` instead of the bitwise operator `&`.

Reply:
Nice catch, thanks!